gpytorch/lazy/lazy_tensor.py

```diff
@@ -400,7 +398,8 @@ def _cholesky(self):
         if evaluated_mat.requires_grad:
             evaluated_mat.register_hook(_ensure_symmetric_grad)
```

Comment:
@jacobrgardner looks like this has been merged in. Should we remove this?

Reply:
@vishwakftw, have all outstanding issues with the symmetric derivative been resolved?

Reply:
@Balandat yes, as far as I am aware, symmetric derivative issues have been resolved.

Reply:
great, I'll remove that hook in this PR as well then.

Reply:
hmm, actually, removing this causes more test failures than I have time to fix :)